contracts/test/v0.8/foundry/vrf/VRFV2Plus.t.sol

```diff
@@ -248,10 +249,23 @@ contract VRFV2Plus is BaseTest {
       extraArgs: VRFV2PlusClient._argsToBytes(VRFV2PlusClient.ExtraArgsV1({nativePayment: true}))
     });
     (, uint96 ethBalanceBefore, , , ) = s_testCoordinator.getSubscription(subId);
-    vm.expectEmit(true, true, false, true);
+
     uint256 outputSeed = s_testCoordinator.getRandomnessFromProofExternal(proof, rc).randomness;
-    emit RandomWordsFulfilled(requestId, outputSeed, subId, 123150, rc.extraArgs, true);
+    vm.recordLogs();
     s_testCoordinator.fulfillRandomWords{gas: 1_500_000}(proof, rc);
+    VmSafe.Log[] memory entries = vm.getRecordedLogs();
+    assertEq(entries[0].topics[1], bytes32(uint256(requestId)));
+    assertEq(entries[0].topics[2], bytes32(uint256(subId)));
+    (uint256 loggedOutputSeed, uint256 loggedPayment, bytes memory loggedExtraArgs, bool loggedSuccess) = abi.decode(
+      entries[0].data,
+      (uint256, uint256, bytes, bool)
+    );
+    assertEq(loggedOutputSeed, outputSeed);
+    assertGe(loggedPayment, 110000);
+    assertLe(loggedPayment, 150000);
```

Comment:
I think this would still be flaky for different optimizer runs no?

Reply:
Sure, removing this